Makefile.am

```diff
@@ -1,7 +1,8 @@
 SUBDIRS = server client data data/init.d data/systemd
 ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
-EXTRA_DIST = autogen.sh README.md hatohol.spec
+EXTRA_DIST = autogen.sh README.md hatohol.spec \
+			 COPYING.GPLv3 COPYING.LGPLv3
```

Comment:
How many I should use tab?
If project-hatohol decided coding rules, please tell me it.